### PR TITLE
chore(content-cover): move content cover component

### PR DESCRIPTION
**Pull request by zwaardje**
Move content cover component to better structure

### CODE REVIEW

File: src/js/components/organisms/content-cover/content-cover.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+@WithRender
+@Component({
+    name: 'ContentCover',
+})
+
+export class ContentCover extends Vue implements ContentCoverProps {
+    @Prop({ type: String, required: true }) title!: ContentCoverProps['title'];
+    @Prop({ type: String, required: true }) description!: ContentCoverProps['description'];
+    @Prop({ type: Object }) cta?: ContentCoverProps['cta'] | undefined;
```
Comment (matijs) [inline src/js/components/organisms/content-cover/content-cover.ts]:
```suggestion
    @Prop({ type: Object }) cta?: ContentCoverProps['cta'];
```

File: src/js/components/organisms/content-cover/content-cover.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    @Prop({ type: Object }) image?: ContentCoverProps['image'] | undefined;
```
Comment (matijs) [inline src/js/components/organisms/content-cover/content-cover.ts]:
```suggestion
    @Prop({ type: Object }) image?: ContentCoverProps['image'];
```

File: src/js/components/organisms/content-cover/content-cover.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    getClassNames(type): string {
```
Comment (matijs) [inline src/js/components/organisms/content-cover/content-cover.ts]:
```suggestion
    getClassNames(type: string): string {
```

File: src/js/components/organisms/content-cover/content-cover.unit.spec.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
+/** @format */
```
Comment (matijs) [inline src/js/components/organisms/content-cover/content-cover.unit.spec.ts]: This whole file doesn't add anything and should be removed from the hygen generator.

File: src/js/components/organisms/content-cover/index.ts [hunk 5]
```diff
@@ -0,0 +1,3 @@
+/** @format */
+
+export { ContentCover as default } from './content-cover';
```
Comment (matijs) [inline src/js/components/organisms/content-cover/index.ts]: Fine with me, but deviates from the default.

File: src/js/components/organisms/content-cover/content-cover.unit.spec.ts (hunk 4 above)
Comment (zwaardje) [reply src/js/components/organisms/content-cover/content-cover.unit.spec.ts]: Basic unit test, not going to remove that.

**Pull request merged**